service/controller/release/resource/status/cluster.go

```diff
@@ -41,10 +42,8 @@ func consolidateClusterVersions(clusters []tenantCluster) (releaseVersions map[s
 // Returns a list of tenant clusters currently running on the installation.
 func (r *Resource) getCurrentTenantClusters(ctx context.Context) ([]tenantCluster, error) {
 	tcGetters := []func(context.Context) ([]tenantCluster, error){
+		r.getCAPIClusters,
 		r.getCurrentAWSClusters,
-		r.getCurrentAzureClusters,
-		r.getLegacyAWSClusters,
-		r.getLegacyAzureClusters,
```

Comment:
Legacy azure clusters don't exist anymore. Current azure clusters use CAPI `Cluster` CR so no need to find them separately.